### ISSUE TITLE
Duplications and Combinations

### ISSUE THREAD

**Issue by DiamondBoyy** [HashLips/generative-art-node#35]
Anyone know how to make the output have no duplications. Can someone help me with the code pls

**Comment by cfech** [comment]
I do not believe based on this code that it is possible to ensure no duplications.
